README.md

```diff
@@ -56,8 +56,8 @@ Typical usage would be something like this:
 import project from '@dojo/web-editor/project';
 
 (async () => {
-	await project.load('some.project.json');
-	console.log('Loaded!');
+    await project.load('some.project.json');
```

Comment:
That was intentional... tabs are just causing problems in the README from spacing, when it gets rendered it looks totally unnatural and unreadable... 😭